options/locale/locale_en-US.ini

```diff
@@ -3354,6 +3357,9 @@ create_branch = created branch <a href="%[2]s">%[3]s</a> in <a href="%[1]s">%[4]
 starred_repo = starred <a href="%[1]s">%[2]s</a>
 watched_repo = started watching <a href="%[1]s">%[2]s</a>
 
+empty = No activities
+empty.desc = You can choose to <a  rel="noopener noreferrer" href="%s">create a repository </a>, <a  rel="noopener noreferrer" href="%s">migrate an external repository</a>, or <a  rel="noopener noreferrer" href="%s">create an organization</a>
```

Comment:
Internal links don't need these `rel` attributes. They are only for external ones.